src/main/resources/hudson/plugins/collabnet/orchestrate/BuildNotifier/config.jelly

```diff
@@ -17,21 +17,23 @@
   </f:optionalBlock>
 
   <f:block>
-      <f:optionalBlock name="webhook" title="Notify TeamForge" checked="${instance.webhookUrl != null}">
+      <f:radioBlock name="config" value="webhook" title="Notify TeamForge" checked="${instance.webhookUrl != null}">
```

Comment:
This is a behavior change. Existing version supports sending notification to both TeamForge and EventQ. What happens if there are customers using it?

Reply:
Nope. Existing plugins support either Teamforge or EventQ to send data. If you choose both, there will be a message in the console output of the job as below
"Build information NOT sent: Can't send notification to TeamForge and EventQ at the same time. Please use either TeamForge/EventQ."
To avoid the misunderstanding, we changed the check box to radio button for better usability.

Reply:
Please create a separate pull request for this issue